tests/core/gui/test_recovery.py

```diff
@@ -110,19 +112,6 @@ def _error_shown():
 
             await aqtbot.wait_until(_error_shown)
 
-    elif kind == "already_exists":
```

Comment:
I'd rather we wrote something in the file initialy, go through the recovery device creation, and check that the file has been overwritten instead of deleting this part of the test.